Лабораторная работа 2/task_1.py

```diff
@@ -0,0 +1,15 @@
+salary = 5000  # Ежемесячная зарплата
+spend = 6000  # Траты за первый месяц
+months = 10  # Количество месяцев, которое планируется протянуть без долгов
+increase = 0.03  # Ежемесячный рост цен #В условии сказано, что increase 5%
+
+# TODO Рассчитайте подушку безопасности, чтобы протянуть 10 месяцев без долгов
+
+new_spend = spend
+all_salary = salary * months
+for _ in range(months - 1):
```

Comment:
```suggestion
for _ in range(months):
```
В этом случае как надо изменить ваш код, чтобы правильно работало